DNN Platform/Modules/ResourceManager/App_LocalResources/ResourceManager.resx

```diff
@@ -411,4 +411,13 @@
   <data name="Download.Text" xml:space="preserve">
     <value>Download</value>
   </data>
+  <data name="Upload.Text" xml:space="preserve">
+    <value>Upload</value>
+  </data>
+  <data name="ExtractUploads.Text" xml:space="preserve">
+    <value>Extract uploaded archives</value>

```

Comment:
```suggestion
    <value>Extract uploaded ZIP files</value>
```
Not sure if this is better or too long, but I think using "ZIP" makes it clearer.

Reply:
Alternate...
```suggestion
    <value>Extract ZIPs</value>
```